src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt

```diff
@@ -150,7 +150,7 @@ abstract class Address(
   SPECIAL_NEEDS_CODE - always null
   CITY_NAME - always null
    */
-) : NomisAuditableEntityWithStaff() {
+) : NomisAuditableEntityBasic() {
```

Comment:
Nobody is using the staff display name - so no need to join on staff user table.